### DISCUSSION TITLE
How to show my wiki articles in two columns?

### DISCUSSION THREAD

**Discussion by classicus69** [datenstrom/community#397] [Ask a question] [answered]
Hi 

Firstly congrats on a great system!

I wonder if you can help me, I would like to present my wiki articles in columns like https://mayberg.se
Can someone help please. Something where we can define number of titles per column and number of columns

Thanks
Best

**Comment by schulle4u** [comment]
You can use CSS properties in your theme for any of your div elements to have multiple list columns, e.g. for the `wikipages` like on the Mayberg website. A comprehensive chapter with usage examples can be found [at W3Schools](https://www.w3schools.com/css/css3_multiple_columns.asp). 

HTH
Steffen

**Comment by classicus69** [comment]
Thank you for coming back to me Steffen. But wont I need php to split the data... if the rows reach 5 then create second column ... etc? Or are Mayberg using shortcuts with limits? Eg. First colum show first 5 titles, secomf column show next 5 titles, etc.

Appreciate the help.

Keep up the good work.

Best regards
Luis

**Comment by classicus69** [comment]
something like this maybe?https://stackoverflow.com/questions/21263798/split-query-results-in-2-divs#comment32036864_21263798thanksluisOn 11 Feb 2019 15:02, Steffen Schultz <notifications@github.com> wrote:You can use CSS properties in your theme for any of your div elements to have multiple list columns, e.g. for the wikipages like on the Mayberg website. A comprehensive chapter with usage examples can be found at W3Schools.
HTH
Steffen

**Comment by schulle4u** [comment, marked as answer]
From what I can see Mayberg only uses CSS properties for the multi-column lists on their site: 

```
.content .wikipages ul{
  column-count: 2; 
  padding: 0;
  list-style: none
}
```

It seems this only allows to specify the column count, but not the item count in a column. Probably the behaviour of such elements also depend on screen size and resolution. On my screen the 9 articles from the first list split up into 5 on the left, 4 on the right column, the second list split into 2/2. I'm no CSS expert, maybe someone can tell you more. 😀
